src/App.jsx

```diff
@@ -1,107 +1,28 @@
 import './App.scss';
+import { PostList } from './components/PostList';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+const getUserById = userId => usersFromServer.find(user => user.id === userId)
+      || null;
+// eslint-disable-next-line
+const getCommentById = commentId => commentsFromServer.filter(comment => comment.postId === commentId)
```

Comment:
no need to return default `null` if there will not be filtered any comment, you return empty array, below you check if array is not empty